File: LP/c/1.c
#include <stdio.h>
#include <math.h>

int main(){

    double n,x;
    printf("informe um inteiro: "); scanf("%lf", &x);
    printf("informe um inteiro não-negativo: "); scanf("%lf", &n);
    printf("resultado = %lf", pow(x, n));

    return 0;
}

File: LP/go/1.go
package main
import "fmt"

func main() {
    x, n := 0,0
    fmt.Println("Informe um inteiro: ")
    fmt.Scanf("%d",&x)
    result := x
    fmt.Println("Informe um inteiro positivo: ")
    fmt.Scanf("%d",&n)
    for i:=1; i<n; i++{
        result *= x
    }
    fmt.Println("resultado = ", result)
}
